### PR TITLE
{lang} [intel-2015a] python 3.4.3

### PR DESCRIPTION
**Pull request by JensTimmerman**

### CODE REVIEW

File: easybuild/easyconfigs/p/Python/Python-3.4.3-intel-2015a.eb [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+version = '3.4.3'
+
+homepage = 'http://python.org/'
+description = "Python is a programming language that lets you work more quickly and integrate your systems more effectively."
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+toolchainopts = {'pic': True, 'opt': True, 'optarch': True}
+
+numpyversion = '1.9.1'
+scipyversion = '0.14.1'
```
Comment (boegel) [inline easybuild/easyconfigs/p/Python/Python-3.4.3-intel-2015a.eb]: use latest 0.15.1?

File: easybuild/easyconfigs/p/Python/Python-3.4.3-intel-2015a.eb [hunk 2]
```diff
@@ -0,0 +1,107 @@
+name = 'Python'
+version = '3.4.3'
+
+homepage = 'http://python.org/'
+description = "Python is a programming language that lets you work more quickly and integrate your systems more effectively."
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+toolchainopts = {'pic': True, 'opt': True, 'optarch': True}
+
+numpyversion = '1.9.1'
```
Comment (boegel) [inline easybuild/easyconfigs/p/Python/Python-3.4.3-intel-2015a.eb]: 1.9.2?

File: easybuild/easyconfigs/p/Python/Python-3.4.3-intel-2015a.eb [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+source_urls = ['http://www.python.org/ftp/%(namelower)s/%(version)s/']
+sources = [SOURCE_TGZ]
+
+# python needs bzip2 to build the bz2 package
+dependencies = [
+    ('bzip2', '1.0.6'),
+    ('zlib', '1.2.8'),
+    ('libreadline', '6.3'),
+    ('ncurses', '5.9'),
+#   ('OpenSSL', '1.0.1k'),  # OS dependency should be preferred if the os version is more recent then this version, it's
```
Comment (boegel) [inline easybuild/easyconfigs/p/Python/Python-3.4.3-intel-2015a.eb]: 1.0.1m is there, try building it?

File: easybuild/easyconfigs/p/Python/Python-3.4.3-intel-2015a.eb [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+#   nice to have an up to date openssl for security reasons
+]
+
+osdependencies = [('openssl-devel', 'libssl-dev')]
+
+# order is important!
+# package versions updated Jan 19th 2015
+exts_filter = ('PYTHONPATH="%(installdir)s:$PYTHONPATH python -c "import %(ext_name)s"', '')
+exts_list = [
+    ('setuptools', '11.3.1', {
```
Comment (boegel) [inline easybuild/easyconfigs/p/Python/Python-3.4.3-intel-2015a.eb]: 15.2?

File: easybuild/easyconfigs/p/Python/Python-3.4.3-intel-2015a.eb [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+        'source_urls': ['http://pypi.python.org/packages/source/s/setuptools/'],
+    }),
+    ('pip', '6.0.6', {
```
Comment (boegel) [inline easybuild/easyconfigs/p/Python/Python-3.4.3-intel-2015a.eb]: 6.1.1

File: easybuild/easyconfigs/p/Python/Python-3.4.3-intel-2015a.eb [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+        'source_urls': ['http://pypi.python.org/packages/source/p/pip/'],
+    }),
+    ('nose', '1.3.4', {
```
Comment (boegel) [inline easybuild/easyconfigs/p/Python/Python-3.4.3-intel-2015a.eb]: 1.3.6

**Comment by boegel** [comment]
@JensTimmerman: please go through the list of extensions and use the latest versions

File: easybuild/easyconfigs/p/Python/Python-3.4.3-intel-2015a.eb (hunk 3 above)
Comment (JensTimmerman) [inline easybuild/easyconfigs/p/Python/Python-3.4.3-intel-2015a.eb]: but it's commented out?

Comment (boegel) [inline easybuild/easyconfigs/p/Python/Python-3.4.3-intel-2015a.eb]: True, but we try to keep it updated regardless, so people can easily switch to an OS-provided dep is they think it's better.

Comment (JensTimmerman) [inline easybuild/easyconfigs/p/Python/Python-3.4.3-intel-2015a.eb]: Do we, or don't we? This is  confusing to me, if we do, then we shouldn't comment it out, if we don't, then we shouldn't put time in it?

**Comment by JensTimmerman** [comment]
how about we keep this one, and bump all versions when 3.5 comes out?

**Comment by JensTimmerman** [comment]
see also https://github.com/hpcugent/easybuild-easyconfigs/pull/1601 for a python 3.4.x with intel 2015a

File: easybuild/easyconfigs/p/Python/Python-3.4.3-intel-2015a.eb (hunk 3 above)
Comment (boegel) [inline easybuild/easyconfigs/p/Python/Python-3.4.3-intel-2015a.eb]: We bump the version in the commented out `OpenSSL` dep line, and ship the matching easyconfig too. So people don't have to figure out how to get that particular OpenSSL version to build.

**Comment by boegel** [comment]
fine with #1601, I'll update this bugger myself then

**Comment by boegel** [comment]
PR with updated versions in #1602

**Comment by JensTimmerman** [comment]
thx

**Comment by boegel** [comment]
No need to close, this will auto-merge when #1602 is merged, but fine. :-)

**Pull request closed without merging**